.travis.yml

```diff
@@ -4,8 +4,9 @@ python:
   - "2.7"
   - "3.5"
   - "3.6"
+  - "3.7"
 install:
-  - if [[ $TRAVIS_PYTHON_VERSION == 3.5 || $TRAVIS_PYTHON_VERSION == 3.6 ]]; then pip install aiohttp==2.0.6 asynctest==0.10.0; fi
+  - if [[ $TRAVIS_PYTHON_VERSION == 3.5 || $TRAVIS_PYTHON_VERSION == 3.6 || $TRAVIS_PYTHON_VERSION == 3.7 ]]; then pip install aiohttp==3.6.2 asynctest==0.13.0; fi
```

Comment:
We need a newer version of `aiohttp` to support python 3.7 & while I was here I also updated the version of `asynctest` test.